packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -577,7 +577,7 @@ export default class NotificationServicesController extends BaseController<
   #registerMessageHandlers(): void {
     this.messagingSystem.registerActionHandler(
       `${controllerName}:updateMetamaskNotificationsList`,
-      async (...args) => this.updateMetamaskNotificationsList(...args),
+      this.updateMetamaskNotificationsList.bind(this),
```

Comment:
This is equivalent to the previous version. It aligns with the other action handlers.